radix-engine-lib/src/core/mod.rs

```diff
@@ -0,0 +1,5 @@
+pub mod expression;
```

Comment:
Can we move all types in this folder to `engine/types`?

Reply:
`network` and `expression` aren't engine related abstractions so would rather just keep them here until we find a more appropriate place to put these.